lib/rdf2marc/rdf2model/mappers/number_and_code_fields.rb

```diff
@@ -77,7 +77,8 @@ def cataloging_source
         end
 
         def agency
-          agency_uri = item.admin_metadata.query.path_first_uri([BF.source])
+          agency_uri = item.admin_metadata.query.path_first_uri([[BF.assigner, BF.Agent]]) \
```

Comment:
This is sort of a challenge to read, can we just have the `||` at the end of the line and not have `\` at all?

Reply:
Definitely. Fixed.